runtime/proto/corfu_options.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto2";
```

Comment:
so, can you tell me again what's the problem with proto3?

Reply:
This was just to test whether it has an impact on the consumption of corfu_options when the application protobuf is proto 2 or 3. This was initially to allow backward compatibility such that the user can have a proto2 environment and still be able to consume our files. This, however, did not work.
I can change this back to proto3.